x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_events_query.ts

```diff
@@ -23,30 +24,49 @@ interface BuildEventsSearchQuery {
   size: number;
   sortOrder?: estypes.SortOrder;
   searchAfterSortIds: estypes.SortResults | undefined;
-  timestampOverride: TimestampOverrideOrUndefined;
+  primaryTimestamp: TimestampOverride;
+  secondaryTimestamp: TimestampOverrideOrUndefined;
   trackTotalHits?: boolean;
 }
 
+interface BuildEqlSearchRequestParams {
+  query: string;
+  index: string[];
+  from: string;
+  to: string;
+  size: number;
+  filters: FiltersOrUndefined;
+  primaryTimestamp: TimestampOverride;
+  secondaryTimestamp: TimestampOverrideOrUndefined;
+  exceptionLists: ExceptionListItemSchema[];
+  runtimeMappings: estypes.MappingRuntimeFields | undefined;
+  eventCategoryOverride?: string;
+  timestampField?: string;
+  tiebreakerField?: string;
+}
+
 const buildTimeRangeFilter = ({
   to,
   from,
-  timestampOverride,
+  primaryTimestamp,
+  secondaryTimestamp,
 }: {
   to: string;
   from: string;
-  timestampOverride?: string;
+  primaryTimestamp: TimestampOverride;
+  secondaryTimestamp: TimestampOverrideOrUndefined;
 }): estypes.QueryDslQueryContainer => {
   // If the timestampOverride is provided, documents must either populate timestampOverride with a timestamp in the range
   // or must NOT populate the timestampOverride field at all and `@timestamp` must fall in the range.
   // If timestampOverride is not provided, we simply use `@timestamp`
```

Comment:
Can you update this comment to refer to `primaryTimestamp` and `secondaryTimestamp` instead?